pkg/apis/application/v1alpha1/app_types_test.go

```diff
@@ -98,6 +98,11 @@ func newAppExampleCR() *App {
 				Namespace: "f2def",
 			},
 		},
+		NamespaceConfig: AppSpecNamespaceConfig{
+			Labels: map[string]string{
```

Comment:
```suggestion
			Annotations: map[string]string{
```

I like the real world example but it should be an annotation.